sanic/config.py

```diff
@@ -56,76 +58,13 @@ def __getattr__(self, attr):
     def __setattr__(self, attr, value):
         self[attr] = value
 
-    def from_envvar(self, variable_name):
```

Comment:
Do not remove these just yet. We have a breaking change freeze between now and 20.12 LTS. Therefore we still need these old methods to work.

They should (however) throw deprecation warnings.

Reply:
How to "throw deprecation warnings" ?

Reply:
```python
import warnings

warnings.warn(
    "Using `from_envvar` is deprecated and will be removed in v21.3, use `update_config` instead",
    DeprecationWarning,
)
```

https://docs.python.org/3/library/warnings.html

Reply:
Thanks for this info.